### DISCUSSION TITLE
Can I use openssl 1.1.1v to compile Win32-OpenSSH, how to compile？

### DISCUSSION THREAD

**Discussion by Mr-LZG** [PowerShell/Win32-OpenSSH#2133] [Q&A]
Can I use openssl 1.1.1v to compile Win32-OpenSSH, how to compile？

**Comment by vthiebaut10** [comment]
I have never tried it myself, but I believe that you could do it. You can try building the libcrypto.dll from OpenSSL, and then modify each of the Visual Studio projects in the OpenSSH solution to use that instead of the LibreSSL. But again, I have never tried doing it myself.

**Reply by RappelBerryPi** [reply]
I realize that the question was about libreSSL 1.1.1. Since the question has been asked openSSL and LibreSSL are 3.0+ and are "incompatible" as they change over time. 

Is the answer the same as above? I ask because on every Windows system I've used it's always been LibreSSL 3.0+. 

Also, why was LibreSSL chosen over OpenSSL?

As a follow up to that question, the libSSL and libCrypto are in System32 as DLLs. Could you compile OpenSSL into the dlls, and then just drop those into the System32 folder, or is OpenSSH on Windows statically compiled?

Thank you!
